clientd3d/d3drender_skybox.h

```diff
@@ -14,9 +14,30 @@
 #ifndef _D3DRENDERSKYBOX_H
 #define _D3DRENDERSKYBOX_H
 
-void D3DRenderSkyBoxBegin();
-void D3DRenderSkyBox(room_type* room, Draw3DParams* params, room_contents_node* pRNode, 
-	int& angleHeading, int& anglePitch);
+struct SkyboxRenderParams {
+    LPDIRECT3DVERTEXDECLARATION9 decl1dc;
+    d3d_driver_profile gD3DDriverProfile;
```

Comment:
"g" is a prefix meaning global variable, which this no longer is.  Should rename these.